### PR TITLE
Release 1.7.0 Beta

### PR DESCRIPTION
**Pull request by vinnyjth**
### What does this PR do, or why is it needed?

This PR lays the groundwork for the 1.7.0 upgrade. It doesn't enhance any features specifically, but does move us towards being able to update the client and start unlocking some of the new features in 1.6 / 1.7 as well as integrating Core prayer.

### How do I test this PR?

Boot up the current `corinth` app and click around. 

## TODO

- [ ] I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))
- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] No new warnings
- [ ] Upload Screenshots/GIF(s) if applicable
- [ ] Set a relevant reviewer

## REVIEW

- [ ] Review updates to test coverage and snapshots
- [ ] Review code through the lens of being concise, simple, and well-documented

---

> The purpose of a PR Review is to _improve the quality of the software._

### CODE REVIEW

**Review by briankalwat — CHANGES_REQUESTED** [review]
@vinnyjth seeing a few issues initially while running these changes, specifically with the home feed:
- New feature block directly above the events block.
image
- New feature block directly after next steps block that looks like it includes all home feed content items.
image

**Comment by vinnyjth** [comment]
@briankalwat Thanks Brian! I think I fixed that in a commit that you must not have locally. Can you try pulling the latest code and give it another shot? 

Screen Shot 2020-11-04 at 8 19 08 AM

**Review by briankalwat — DISMISSED** [review]
Looks like everything appears normal now.

File: config.yml
Comment (vinnyjth) [inline config.yml:345]:
```suggestion
        name: Stories        
```
typo

**Comment by vinnyjth** [comment]
@briankalwat Can you please re-approve? I believe I dismissed your review by fixing a typo :)

**Review by briankalwat — APPROVED** [review]

**Comment by vinnyjth** [comment]
@briankalwat Okay, I'll need your approval one more time. I had to push some more test to make sure we are at 100% coverage. 

I think we could look at turning off "stale review dismissal" if that's okay with you. I think you can do it from the branch protection settings.

**Review by briankalwat — APPROVED** [review]

**Pull request merged**
